framework/src/controller/schema/application.js

```diff
@@ -13,6 +13,211 @@ module.exports = {
 	constants: {
 		id: '#/app/constants',
 		type: 'object',
+		required: [
+			'ACTIVE_DELEGATES',
+			'BLOCK_SLOT_WINDOW',
+			'ADDITIONAL_DATA',
+			'BLOCK_RECEIPT_TIMEOUT',
+			'EPOCH_TIME',
+			'FEES',
+			'MAX_PAYLOAD_LENGTH',
+			'MAX_PEERS',
+			'MAX_SHARED_TRANSACTIONS',
+			'MAX_TRANSACTIONS_PER_BLOCK',
+			'MAX_VOTES_PER_TRANSACTION',
+			'MAX_VOTES_PER_ACCOUNT',
+			'MIN_BROADHASH_CONSENSUS',
+			'MULTISIG_CONSTRAINTS',
+			'NETHASHES',
+			'NORMALIZER',
+			'REWARDS',
+			'TOTAL_AMOUNT',
+			'UNCONFIRMED_TRANSACTION_TIMEOUT',
+			'EXPIRY_INTERVAL',
+		],
+		properties: {
+			ACTIVE_DELEGATES: {
+				type: 'integer',
+				min: 0,
+			},
+			BLOCK_SLOT_WINDOW: {
+				type: 'integer',
```

Comment:
We had tested all of our application as per `5` second slow window. I think we should set it as a `min` limit. Unless we successfully test our blockchain with lower slot window we should not allow users to do it. 

@MaciejBaj Please confirm.